db/queries.py

```diff
@@ -3,44 +3,78 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    #  used `select_related` to reduce the load on the database for example
+    return list(
+        Message.objects.filter(
+            text__icontains=word
+        ).select_related("chat", "user")
+    )
```

Comment:
No need to use select_related here, cause we wont to check the fields of related objects

Reply:
ok)